notebooks/en/index.md

```diff
@@ -12,6 +12,7 @@ Check out the recently added notebooks:
 - [Fine-tuning SmolVLM using direct preference optimization (DPO) with TRL on a consumer GPU](fine_tuning_vlm_dpo_smolvlm_instruct)
 - [Smol Multimodal RAG: Building with ColSmolVLM and SmolVLM on Colab's Free-Tier GPU](multimodal_rag_using_document_retrieval_and_smol_vlm)
 - [Fine-tuning SmolVLM with TRL on a consumer GPU](fine_tuning_smol_vlm_sft_trl)
+- [Multi-Agent Order Management System with MongoDB](mongodb_smolagents_multi_micro_agents)
```

Comment:
Put this at the top since we show the most recent notebooks first, and then remove "Fine-tuning SmolVLM with TRL on a consumer GPU" to keep it tidy :)